socorro/tests/external/es/test_super_search_fields.py

```diff
@@ -218,8 +217,7 @@ def test_validate_super_search_fields(name, properties):
         if properties.get("destination_keys"):
             for key in properties["destination_keys"]:
                 possible_keys = [
-                    # Old keys we're probably migrating from
-                    f"raw_crash.{properties['in_database_name']}",
```

Comment:
None of the keys start with `raw_crash` and we want to make sure that's true going forward. We only want to index values that have been normalized and validated and those are in the processed crash.